src/minicraft/item/Item.java

```diff
@@ -29,17 +29,22 @@ protected Item(String name, Sprite sprite) {
 		this.sprite = sprite;
 	}
 	
-	/// TODO this method (and Menu.renderItemList) is actually slowly getting depricated; I just haven't gotten around to updating all the menus yet.
-	/** Renders an item (sprite & name) in an inventory */
-	public void renderInventory(Screen screen, int x, int y, boolean ininv) {
-		String dispName = getDisplayName();
-		sprite.render(screen, x, y);
-		if(ininv) {
+	// TODO this method (and Menu.renderItemList) is actually slowly getting deprecated; I just haven't gotten around to updating all the menus yet.
+	public void renderInventory(Screen screen, int x, int y, boolean ininv) { 
+		if (ininv) {
+			String dispName = getDisplayName();
+			sprite.render(screen, x, y);
 			String shortname = dispName.length() > 20 ? dispName.substring(0, 20) : dispName;
 			Font.draw(shortname, screen, x + 8, y, Color.WHITE);
+		} else

```

Comment:
is it just me or is there a missing bracket here?

Reply:
No, that's the end of the function.